### PR TITLE
Create separate UAA user accounts and renamed fixtures

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
Tests that login to response-operations-ui/response-operations-social-ui apps use the same uaa account which can cause problems when running in parallel.

# What has changed
Tests now create their own unique uua account. Also, fixtures have been renamed to improve readability

# How to test?
make acceptance_tests

# Links
[Trello](https://trello.com/c/cXUAjCxj/356-t338-acceptance-tests-uaa-user-per-scenario)

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[T338: Acceptance Tests : uaa user per scenario](https://trello.com/c/cXUAjCxj/356-t338-acceptance-tests-uaa-user-per-scenario)

File: common/internal_utilities.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                             auth=(client_id, client_secret))
+
+    response.raise_for_status()
+
+    return response.json().get('access_token')
+
+
+def create_user(access_token, username, password, email, first_name, last_name, url):
+    user = {
+        "userName": "{}".format(username),
```
Comment (AdamHawtin) [inline common/internal_utilities.py]:
```suggestion
        "userName": username,
```

File: common/internal_utilities.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        "name": {
+            "formatted": "{0} {1}".format(first_name, last_name),
```
Comment (AdamHawtin) [inline common/internal_utilities.py]:
```suggestion
           "formatted": f'{first_name} {last_name}',
```

File: common/internal_utilities.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            "primary": False
+        }],
+        "active": True,
+        "verified": True,
+        "password": password
+    }
+
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json',
+               'Authorization': 'Bearer {}'.format(access_token)}
```
Comment (AdamHawtin) [inline common/internal_utilities.py]:
```suggestion
               'Authorization': f'Bearer {access_token}'}
```

File: common/internal_utilities.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+def generate_access_token(client_id, client_secret, url):
+    headers = {'Content-Type': 'application/x-www-form-urlencoded',
+               'Accept': 'application/json'}
+
+    payload = {'grant_type': 'client_credentials',
+               'response_type': 'token',
+               'token_format': 'opaque'}
+
+    response = requests.post(url.format(url), headers=headers,
```
Comment (AdamHawtin) [inline common/internal_utilities.py]:
```suggestion
    response = requests.post(url=url, headers=headers,
```

File: common/internal_utilities.py (hunk 1 above)
Comment (AdeHarrison) [reply common/internal_utilities.py]: fixed

File: common/internal_utilities.py (hunk 2 above)
Comment (AdeHarrison) [reply common/internal_utilities.py]: fixed

File: common/internal_utilities.py (hunk 3 above)
Comment (AdeHarrison) [reply common/internal_utilities.py]: fixed

File: common/internal_utilities.py (hunk 4 above)
Comment (AdeHarrison) [reply common/internal_utilities.py]: fixed

**Pull request merged**